lib/matplotlib/cm.py

```diff
@@ -102,6 +102,10 @@ class ColormapRegistry(Mapping):
     r"""
     Container for colormaps that are known to Matplotlib by name.
 
+    .. admonition:: Experimental
+
+       This API is experimental and may still change in future versions.
```

Comment:
Add a sentence encouraging feedback here too?

Reply:
I'd rather not. See the comment on feedback above.